plugin/src/App/Components/Studio/StudioPluginAction.lua

```diff
@@ -0,0 +1,40 @@
+local Rojo = script:FindFirstAncestor("Rojo")
+local Plugin = Rojo.Plugin
+
+local Roact = require(Rojo.Roact)
+
+local Dictionary = require(Plugin.Dictionary)
+
+local StudioPluginContext = require(script.Parent.StudioPluginContext)
+
+local e = Roact.createElement
+
+local StudioPluginAction  = Roact.Component:extend("StudioPluginAction")
+
+function StudioPluginAction:init()
+	self.pluginAction = self.props.plugin:CreatePluginAction(
+		self.props.name, self.props.title, self.props.description, self.props.icon, self.props.bindable
+	)
+
+	self.pluginAction.Triggered:Connect(self.props.onTriggered)
+end
+
+function StudioPluginAction:render()
+	return
```

Comment:
```suggestion
	return nil
```

We should explicitly return nil in Roact components that render nothing.